app-emulation/rkt/files/rkt-gc.service

```diff
@@ -1,7 +0,0 @@
-[Unit]
```

Comment:
They were only shipped due to too aggressive timing but these were corrected in rkt: https://github.com/coreos/rkt/commit/00c539e60a6f3b1bfb724f0db0bd1b88359239aa.